orchestration/test/orchestration_gate.cpp

```
@@ -0,0 +1,926 @@
+}
+
+bool containsText(const std::string& hay, const std::string& needle) {
+  return hay.find(needle) != std::string::npos;
+}
+
+}  // namespace
+
+int main(int argc, char** argv) {
+  const std::string root = (argc > 1) ? argv[1] : ".";
+  const std::string scratch = (argc > 2) ? argv[2] : "/tmp";
+  const std::string fx = root + "/orchestration/test/fixtures";
```

Clang (14.0.6)

[warning] variable name 'fx' is too short, expected at least 3 characters

(readability-identifier-length)